contracts/Gauge.sol

```diff
@@ -120,30 +117,6 @@ contract Gauge is BaseGauge, ERC20Upgradeable, IGauge {
             _ve,
             _veYfiRewardPool
         );
-        boostingFactor = 100;
-    }
-
-    /**
-    @notice Set the veYFI token address.
-    @param _veToken the new address of the veYFI token
-    */
-    function setVe(address _veToken) external onlyOwner {
-        require(address(_veToken) != address(0x0), "_veToken 0x0 address");
-        veToken = _veToken;
-        emit UpdatedVeToken(_veToken);
-    }
-
-    /**
-    @notice Set the boosting factor.
-    @dev the boosting factor is used to calculate your boosting balance using the curve boosting formula adjusted with the boostingFactor
-    @param _boostingFactor the value should be between 20 and 500
-    */
-    function setBoostingFactor(uint256 _boostingFactor) external onlyOwner {
```

Comment:
same Q, after we launch and in case of a mistake we can deprecate these gauges and move to a new one i guess is the plan?